### PR TITLE
fix: deprecation warning for contenttypes.generic in Django 1.8

### PR DESCRIPTION
**Pull request by ungikim**
c.f. https://docs.djangoproject.com/fr/1.8/releases/1.7/#reorganization-of-django-contrib-contenttypes

### CODE REVIEW

File: djcelery_model/models.py [hunk 1]
```diff
@@ -2,7 +2,16 @@
...
+try:
+    # Django >= 1.7
+    from django.contrib.contenttypes.fields import GenericForeignKey
+except ImportError:
+    from django.contrib.contenttypes.generic import GenericForeignKey
+try:
+    # Django >= 1.7
+    from django.contrib.contenttypes.fields import GenericRelation
+except ImportError:
+    from django.contrib.contenttypes.fields import GenericRelation
```
Comment (mback2k) [inline djcelery_model/models.py]: I don't see any difference to the import statement within the try-block. Shouldn't the second line also read .generic?

**Comment by ungikim** [comment]
The module django.contrib.contenttypes.generic is deprecated in django 1.7 and will removed in django 1.9. (c.f. https://docs.djangoproject.com/en/dev/releases/1.9/#features-removed-in-1-9 ).
There is no .fields module on <= 1.6.
That code is for compatibility with >= 1.7 and not (<= 1.6).

File: djcelery_model/models.py (hunk 1 above)
Comment (ungikim) [inline djcelery_model/models.py]: The module django.contrib.contenttypes.generic is deprecated in django 1.7 and will removed in django 1.9. (c.f. https://docs.djangoproject.com/en/dev/releases/1.9/#features-removed-in-1-9 ).
There is no .fields module on <= 1.6.
That code is for compatibility with >= 1.7 and not (<= 1.6).

Comment (cabello) [inline djcelery_model/models.py]: I agree it should be `generic`. Django 1.6.11 has no `fields` module just the `generic`: https://github.com/django/django/blob/1.6.11/django/contrib/contenttypes/generic.py

Comment (mback2k) [inline djcelery_model/models.py]: Would you mind updating this pull request or creating a new one for this? Thanks.

Comment (mback2k) [inline djcelery_model/models.py]: Thanks @cabello, I just used and comitted your proposal.

**Comment by mback2k** [comment]
Close, since the changes by @ungikim, @mbelousov and @cabello have been merged.

**Pull request closed without merging**